src/utilities/findBreakingChanges.js

```diff
@@ -105,6 +110,80 @@ export function findTypesThatChangedKind(
   return breakingChanges;
 }
 
+/**
+ * Given two schemas, returns an Array containing descriptions of any breaking
+ * changes in the newSchema related to removal of an arg.
+ */
+export function findBreakingArgChanges(
+ oldSchema: GraphQLSchema,
+ newSchema: GraphQLSchema
+): Array<BreakingChange> {
+  const oldTypeMap = oldSchema.getTypeMap();
+  const newTypeMap = newSchema.getTypeMap();
+
+  const breakingArgChanges = [];
+
+  Object.keys(oldTypeMap).forEach(typeName => {
+    const oldType = oldTypeMap[typeName];
+    const newType = newTypeMap[typeName];
+    if (
+      !(oldType instanceof GraphQLObjectType) ||
+      !(newType instanceof oldType.constructor)
+    ) {
+      return;
+    }
+
+    const oldTypeFields: GraphQLFieldMap<*, *> = oldType.getFields();
+    const newTypeFields: GraphQLFieldMap<*, *> = newType.getFields();
+
+    Object.keys(oldTypeFields).forEach(fieldName => {
+      if (!newTypeFields[fieldName]) {
+        return;
+      }
+
+      oldTypeFields[fieldName].args.forEach(oldArgDef => {
+        const newArgs = newTypeFields[fieldName].args;
+        const newTypeArgIndex = newArgs.findIndex(
+          arg => arg.name === oldArgDef.name
+        );
+        const newArgDef = newArgs[newTypeArgIndex];
+
+        // Arg not present
+        if (newTypeArgIndex < 0) {
+          breakingArgChanges.push({
+            type: BreakingChangeType.ARG_REMOVED,
+            description: `${oldType.name}.${fieldName} arg ` +
+              `${oldArgDef.name} was removed`,
+          });
+
+        // Arg changed type in a breaking way
+        } else if (
+          oldArgDef.type !== newArgDef.type &&
+          getNullableType(oldArgDef.type) !== newArgDef.type
+        ) {
+          breakingArgChanges.push({
+            type: BreakingChangeType.ARG_REMOVED,
```

Comment:
Seems like this should be `ARG_CHANGED_KIND`, not `ARG_REMOVED`, and the tests will need to be updated too.